primitives/src/v1.rs

```diff
@@ -175,6 +175,11 @@ pub const PARACHAINS_INHERENT_IDENTIFIER: InherentIdentifier = *b"parachn0";
 /// The key type ID for parachain assignment key.
 pub const ASSIGNMENT_KEY_TYPE_ID: KeyTypeId = KeyTypeId(*b"asgn");
 
+/// Maximum compressed code size we support right now.
+/// At the moment we have runtime upgrade on chain, which restricts scalability severly. If we want
+/// to have bigger values, we should fix that first.
+pub const MAX_CODE_SIZE: u32 = 3 * 1024 * 1024;
+
```

Comment:
@rphmeier what do you think about this? I guess it'd have to be the upper limit for every chain that will have parachains?

Reply:
I'd suggest making it equal to whatever we have in #2854

Reply:
https://github.com/paritytech/polkadot/blob/0f3a21cb7d99027cb562f116f4ba925625091541/node/service/src/chain_spec.rs#L898